website/notifications/listeners.py

```diff
@@ -1,12 +1,27 @@
+import logging
+from website.notifications.exceptions import InvalidSubscriptionError
 from website.notifications.utils import subscribe_user_to_notifications
 from website.project.signals import contributor_added, project_created
 from website.project.views.contributor import notify_added_contributor
 
+logger = logging.getLogger(__name__)
+
 @project_created.connect
 def subscribe_creator(node):
-    subscribe_user_to_notifications(node, node.creator)
+    try:
+        subscribe_user_to_notifications(node, node.creator)
+    except InvalidSubscriptionError as err:
+        user = node.creator._id if node.creator else 'None'
+        logger.warn('Skipping subscription of user {} to node {}'.format(user, node._id))
+        logger.warn('Reason: {}'.format(str(err)))
 
 @contributor_added.connect
 def subscribe_contributor(node, contributor, auth=None, *args, **kwargs):
-    subscribe_user_to_notifications(node, contributor)
-    notify_added_contributor(node, contributor, auth)
+    try:
+        subscribe_user_to_notifications(node, contributor)
+    except InvalidSubscriptionError as err:
+        user = contributor or 'None'
```

Comment:
Switched is_bookmark_collection to is_collection
removed message_short and message_long
removed user = contributor or 'None'
Ready for further review